package.json

```diff
@@ -82,6 +87,7 @@
     "mini-css-extract-plugin": "^0.4.4",
     "node-sass": "^4.9.4",
     "nodemon": "^1.18.9",
+    "npm-run-all": "^4.1.5",
```

Comment:
this module does the work easily?

Reply:
It helps runs `nodemon` in parallel with two `webpack` watchers (watching for code changes) on the frontend and backend code. Whenever the changes happen, we re-build then nodemon restarts. You will still have to refresh on the frontend